Tools/autotest/autotest.py

```diff
@@ -444,6 +444,7 @@ def run_tests(steps):
     parser.add_option("--map", action='store_true', default=False, help='show map')
     parser.add_option("--experimental", default=False, action='store_true', help='enable experimental tests')
     parser.add_option("--timeout", default=3000, type='int', help='maximum runtime in seconds')
+    parser.add_option("--speedup", default=10, type='int', help='speedup to runthe simulations at')
```

Comment:
runthe -> run the

I don't have a better suggestion, but having a default here will override the defaults in vehicles.

Reply:
I'm fine with the override if your lazy as long as nothing breaks. (Mostly cause I didn't have a better version, and didn't think it was important enough to focus on) As far as the CI side goes we call autotest for each target so we can pass different values there as needed to speed it up.